orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/purple/HotspotType.java

```diff
@@ -1,6 +1,7 @@
 package com.hartwig.hmftools.datamodel.purple;
 
-public enum Hotspot {
+public enum HotspotType
```

Comment:
Suggestion, what about `HotspotProximity` with values INSIDE, NEAR, OUTSIDE

Reply:
I like the proposal for the enum name, but the values could be better. A hotspot is never "wider" than a variant so INSIDE doesn't make sense. Maybe MATCH, NEAR, NON-MATCH would be better.. although I don't like the term NON_MATCH. 

In a way this field captures 2 booleans:
 - Does this variant match with a hotspot?
 - If not, is there a hotspot nearby?

Maybe we should model this more explicitly... the benefit of the current enum though is that the 2nd boolean is irrelevant if the first boolean is true.